src/screentovnc.cpp

```diff
@@ -167,7 +191,7 @@ ScreenToVnc::ScreenToVnc(QObject *parent, bool smoothScaling, float scalingFacto
     char *argv[0];
     m_server = rfbGetScreen(0,argv, m_screen_width, m_screen_height, 8, 3, m_screen->depth() / 8);
```

Comment:
old code nit: this bogus argv thing here feels so wrong... if rfbGetScreen(0, nullptr, ...) is not okay, then char *argv[1] = { nullptr }; would be more as expected...

Reply:
Though it's given explicit argc of 0. Dunno, maybe I'll avoid touching now.